satellitevu/apis/otm.py

```diff
@@ -328,6 +328,29 @@ def get_order(self, *, contract_id: Union[UUID, str], order_id: Union[UUID, str]
         )
         return response.json()
 
+    def cancel_order(
+        self, *, contract_id: Union[UUID, str], order_id: Union[UUID, str]
+    ):
+        """
+        Cancel an order with a given order_id.
+
+        Args:
+            contract_id: Associated ID of the Contract under which the tasking
+            order is stored.
+
+            order_id: UUID representing the order id e.g.
+            "2009466e-cccc-4712-a489-b09aeb772296".
+
+        Returns:
+            None
+        """
+        response = self.make_request(
+            method="POST",
+            url=self.url(f"{str(contract_id)}/tasking/orders/{str(order_id)}/cancel"),
+        )
+        if response.status != 204:
+            raise Exception(f"Error - {response.status} : {response.text}")
```

Comment:
it's not advisable to raise Exception - it's the most "meaningless" exception. I would recommend to at least use [RuntimeError](https://docs.python.org/3/library/exceptions.html#RuntimeError) or define a custom `OtmError(Exception)` or even an `OtmCancelError(OtmError)` to give the actual raised exception a semantic meaning.